templates/predict.ipynb

```diff
@@ -21,34 +21,27 @@
    ],
```

Comment:
<p>What do these plots end up looking like?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/cell_classification/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Plots look like this:
![NimbusViewerWidget](https://github.com/angelolab/cell_classification/assets/22618181/af334e51-f232-4155-af3c-eccece7f6ac8)

Reply:
<p>Looks great!</p>